### PR TITLE
I have some new fix and feature

### PR DESCRIPTION
**Pull request by bjrxyz**
Hi:
1. I fix the insert position. before i fix ,it's in the line 2 and column 2. 
2. I add a feature, automatic insert date by ${date}.

### CODE REVIEW

**Comment by Gigabyte-Giant** [comment]
Greetings, and thanks for the pull request.

I really like that you added a date flag, that's something that I was going to do originally, but didn't.

I purposely left out the new line at the end of comments, because I feel that's something better suited in a comment template.

I'm away from my desk as of now, so I can't review your changes, but I'll try and review them shortly.

Thanks again for the pull request!

File: src/commands.ts [hunk 1]
```diff
@@ -32,9 +32,10 @@ export function insertFileHeaderComment() {
             var template = (langConfs.get(languageStr + ".template") as Array<String>).join("\n");
             
             _editor.edit((edit) => {
-                edit.insert(new vscode.Position(1, 1), template
+                edit.insert(new vscode.Position(0, 0), template
                     .replace("$(projectName)", values.projectName)
-                    .replace("$(currentFile)", values.currentFile));
+                    .replace("$(currentFile)", values.currentFile)
+                    .replace("$(date)", (new Date()).toLocaleString())+"\n");
```
Comment (Gigabyte-Giant) [inline src/commands.ts]: I really think it'd be best to not include the newline in the text that is inserted. Could you remove it for now?

**Comment by Gigabyte-Giant** [comment]
Everything looks good. If you could just remove the newline from the text that gets inserted, that'd be great.

File: src/commands.ts (hunk 1 above)
Comment (bjrxyz) [inline src/commands.ts]: of course

**Comment by bjrxyz** [comment]
Follow your advice，l remove the newline，

**Comment by Gigabyte-Giant** [comment]
Awesome! I'll try and push the update to the extension gallery within the next week!

Thanks again for the pull request!

**Pull request merged**